data/population.json

```diff
@@ -1048,5 +1048,5 @@
       "CountryName": "Zimbabwe",
       "Population": 14439018
     }
-  }
+  
```

Comment:
There are quite some keys in this file that we don't use which is a bit wasteful, could we optimize this somehow?
e.g.:
```json
    ":::Early-demographic dividend": {
      "CountryName": "Early-demographic dividend",
      "Population": 3249140605
    },
    ":::East Asia & Pacific": {
      "CountryName": "East Asia & Pacific",
      "Population": 2328220870
    },
    ":::East Asia & Pacific (excluding high income)": {
      "CountryName": "East Asia & Pacific (excluding high income)",
      "Population": 2081651801
    },
    ":::East Asia & Pacific (IDA & IBRD countries)": {
      "CountryName": "East Asia & Pacific (IDA & IBRD countries)",
      "Population": 2056064424
    },
```

Reply:
yup, will update